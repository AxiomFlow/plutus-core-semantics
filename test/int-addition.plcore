(define result (builtin addInt (2 2)))
