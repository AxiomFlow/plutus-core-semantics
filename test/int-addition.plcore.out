<T> <k> .DefList </k> <env> result |-> 4 </env> </T>
